### PR TITLE
Adds openSUSE Leap SP migration

### PR DESCRIPTION
**Pull request by keichwa**
~***DO NOT MERGE - DOCS REPO IS FROZEN***~

https://github.com/SUSE/spacewalk/issues/13498

# Description

Migrate openSUSE Leap with SP migration; target either next Leap version or SLE as a client.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/client-configuration/pages/client-upgrades-sp-migration.adoc [hunk 1]
```diff
@@ -1,23 +1,31 @@
 [[client-upgrades-spmigration]]
 = Service Pack Migration
 
-Service Pack Migration (SP Migration) allows you to upgrade SLE-based client systems from an SP level to a later one.
+Service Pack Migration (SP Migration) allows you to upgrade SLE-based client systems from an SP level to a later one, for example:
+
+* {sle}{nbsp}15{nbsp}SP1 to SP2
```
Comment (Loquacity) [inline modules/client-configuration/pages/client-upgrades-sp-migration.adoc:6]:
```suggestion
Service Pack (SP) migration allows you to upgrade SLE-based client systems from an SP level to a later one.
For example, you can migrate {sle}{nbsp}15{nbsp}SP1 to {sle}{nbsp}15{nbsp}SP2.
```

File: modules/client-configuration/pages/client-upgrades-sp-migration.adoc [hunk 2]
```diff
@@ -1,23 +1,31 @@
...
+
+You can also migrate openSUSE Leap to a later minor version or to the corresponding SLE SP level, for example:
+
+* openSUSE Leap 15.1 to 15.2, or
+* openSUSE Leap 15.1 to {sle} 15 SP1, or
+* openSUSE Leap 15.2 to {sle} 15 SP2
 
 [WARNING]
 ====
 During migration {productname} automatically accepts any required licenses (EULAs) before installation.
```
Comment (Loquacity) [inline modules/client-configuration/pages/client-upgrades-sp-migration.adoc:16]:
```suggestion
During migration, {productname} automatically accepts any required licenses (EULAs) before installation.
```

File: modules/client-configuration/pages/client-upgrades-sp-migration.adoc [hunk 3]
```diff
@@ -38,6 +46,11 @@ Ensure you have a working system backup available, in case of an emergency.
 . From the [guimenu]``Service Pack Migration - Channels`` page select the correct base channel, including ``Mandatory Child Channels`` and any additional ``Optional Child Channels``.
 . OPTIONAL: Check [guimenu]``Allow Vendor Change`` to allow packages that have changed vendors to be installed.
   If this occurs, a notification is shown with details before the migration is started.
+  +
+[IMPORTANT]
+====
+[guimenu]``Allow Vendor Change`` is required if you want to migrate openSUSE Leap to {sle}.
```
Comment (Loquacity) [inline modules/client-configuration/pages/client-upgrades-sp-migration.adoc:52]:
```suggestion
To migrate openSUSE Leap to {sle}, you must check the [guimenu]``Allow Vendor Change`` option.
```

**Review by Loquacity — CHANGES_REQUESTED** [review]

File: modules/client-configuration/pages/client-upgrades-sp-migration.adoc (hunk 1 above)
Comment (keichwa) [reply modules/client-configuration/pages/client-upgrades-sp-migration.adoc:6]: Yes, that's probably the save way.  In the past, we were told that it is possible to skip an SP.  For example, to upgrade from SP0 to SP2.  `zypper migration` and also the Web UI (SP migration) sometimes offered skipping as well.  I do not know whether that is still the case and we want to mention it.  @mcalmer , what do you think, please?

**Review by Loquacity — APPROVED** [review]

File: modules/client-configuration/pages/client-upgrades-sp-migration.adoc [hunk 4]
```diff
@@ -1,23 +1,30 @@
...
+* openSUSE Leap 15.1 to {sle} 15 SP1, or
+* openSUSE Leap 15.2 to {sle} 15 SP2
 
 [WARNING]
 ====
-During migration {productname} automatically accepts any required licenses (EULAs) before installation.
+During migration, {productname} automatically accepts any required licenses (EULAs) before installation.
 ====
 
 In {sle}{nbsp}12 and later, {suse} supports service pack skipping.
```
Comment (keichwa) [inline modules/client-configuration/pages/client-upgrades-sp-migration.adoc:18]:
```suggestion
In {sle}{nbsp}12 and later, {suse} supports service pack skipping if SUSE Customer Center provides it.
```

Comment (Loquacity) [reply modules/client-configuration/pages/client-upgrades-sp-migration.adoc:18]: sure, but let's use the {scc} entity.

**Review by Loquacity — APPROVED** [review]

**Comment by keichwa** [comment]
thanks for all your help, Lana!

**Pull request merged**